CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [v2.3.3] - 15-Feb-2021
+
+### Fixed
+
+- Fixed duplicate alias names returned from namespace endpoint issue.
```

Comment:
- Fixed duplicated status results. The status is retrieved by priority.

This fix has been added https://github.com/nemtech/catapult-rest/issues/496

Reply:
👍